workbench/src/main/settingsStore.js

```diff
@@ -76,6 +76,10 @@ export function setupSettingsHandlers() {
     (event, key) => settingsStore.get(key)
   );
 
+  ipcMain.on(ipcMainChannels.GET_SETTING, (event, key) => {
```

Comment:
I added the `on` listener because I called it with `sendSync` in `HomeTab` (in the `RecentInvestJobs` function). My understanding was it should be synchronous there. Since `electron-store` operations are fast and synchronous, and we usually `await` the result anyway, maybe we should only have the `on` listener?

Reply:
Okay, thanks I missed that `sendSync` call. Hmm this is interesting. `electron-store` operations are synchronous but they run in the `main` process. I think we generally want to avoid synchronous operations in the renderer though, since they can freeze the browser. You're right this might be too fast to even notice, so it's mostly just a design principle. But I'm not sure there's a good reason to break it if we can do `await invoke` instead.